setup.py

```diff
@@ -75,7 +76,7 @@ def generate_build_ext(pkg_dir: str, pkg_name: str):
     # include-folders
     include_dirs = [
```

Comment:
Must these paths be strings, or can we use Path objects?

Reply:
It has to be string.